src/client/Microsoft.Identity.Client/ManagedIdentity/ServiceFabricManagedIdentitySource.cs

```diff
@@ -45,9 +44,9 @@ private ServiceFabricManagedIdentitySource(RequestContext requestContext, Uri en
         {
             _endpoint = endpoint;
             _identityHeaderValue = identityHeaderValue;
-            _userAssignedId = requestContext.ServiceBundle.Config.ManagedIdentityUserAssignedId;
 
-            if (!string.IsNullOrEmpty(_userAssignedId))
+            if (!string.IsNullOrEmpty(requestContext.ServiceBundle.Config.ManagedIdentityUserAssignedClientId) || 
+                !string.IsNullOrEmpty(requestContext.ServiceBundle.Config.ManagedIdentityUserAssignedResourceId))
             {
                 requestContext.Logger.Warning(MsalErrorMessage.ManagedIdentityUserAssignedNotConfigurableAtRuntime);
```

Comment:
What does this warning mean? Is it still relevant? We do add client ID at line 66.

Reply:
This was discussed with Azure SDK team, this is the warning they add for service fabric. The API for service fabric is in preview and it does support user assigned managed identity, see [this](https://learn.microsoft.com/en-us/azure/service-fabric/how-to-deploy-service-fabric-application-user-assigned-managed-identity#application-template). But when we discussed this with Azure SDK team they were not sure and suggested to continue with the warning. Maybe I can add a comment to revisit this warning after we perform our testing or after UAT.